docker/substra-backend/Dockerfile

```diff
@@ -32,8 +36,7 @@ FROM build AS arm64
 RUN apt-get update && apt-get install -y --no-install-recommends \
     # libpq is a required dependency to build psycopg2 from source (support for arm64 platform)
     libpq-dev
-RUN pip3 install --upgrade --no-cache-dir pip==24.0 \
-    && pip3 install --prefix /usr/local --no-cache-dir psycopg2==2.9.9
+RUN pip install --no-cache-dir psycopg2==2.9.9
```

Comment:
Is there a reason why we don't do it on line 19 / 20?

Reply:
Yes, it's only meant to be installed for arm64, it is under `FROM build AS arm64`